client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/components/need-help-link.tsx

```diff
@@ -0,0 +1,14 @@
+import { Button } from '@wordpress/components';
+import { translate } from 'i18n-calypso';
+
+export const NeedHelpLink = ( { onSkip }: { onSkip: () => void } ) => {
```

Comment:
shouldn't the parameter be called `onAction` or `onHelpLinkClicked` or something along those lines? In the context of a separate component it's not "skipping" it's the main action of the component.